README.fr.md

```diff
@@ -1,35 +1,28 @@
-﻿- [English](README.md)
-- [日本語](README.ja.md)
-- [한국어](README.ko.md)
-- [Bahasa Indonesia](README.in.md)
-- [Español](README.es.md)
-- [简体中文](README.zh.md)
-- [français](README.fr.md)
-- [العربية](README.ar.md)
```

Comment:
Pourquoi enlever les liens? C'est probablement mieux de guarder les liens pour les autres langues ici.

Reply:
That reason is #37